source/Esf.Website/App/esfState/esfState.component.html

```diff
@@ -25,8 +42,8 @@
         </div>
         <div class="col-xs-6">
             <label>Result</label>
-            <div class="result-viewer" [text]="queryRunner.queryResult" json-editor [readOnly]="true" style="border: 2px #e7e7e7 solid; display: block; width: 100%; min-height: 200px; overflow: auto;"></div>
-            <div class="alert alert-danger" *ngIf="!!queryRunner.queryError">{{queryRunner.queryError}}</div>
+            <div class="result-viewer" [text]="queryResult" json-editor [readOnly]="true" style="border: 2px #e7e7e7 solid; display: block; width: 100%; min-height: 200px; overflow: auto;"></div>
```

Comment:
I have thoughts aboute the results-viewers and the erros, but we can implement them once we've fully merged your changes